packages/reporter/src/.eslintrc.json

```diff
@@ -3,9 +3,23 @@
     "plugin:@cypress/dev/react",
     "plugin:@cypress/dev/tests"
   ],
+  "parser": "@typescript-eslint/parser",
```

Comment:
good to know we don't need `parser: babel-eslint`. If this works, after this gets merged we can update https://github.com/cypress-io/eslint-plugin-dev/blob/master/lib/index.js#L308 and remove babel-eslint